pkg/network/tracer/offsetguess/tracer.go

```diff
@@ -677,7 +675,7 @@ func (t *tracerOffsetGuesser) checkAndUpdateCurrentOffset(mp *maps.GenericMap[ui
 			if !t.guessTCPv6 && !t.guessUDPv6 {
 				t.logAndAdvance(t.status.Offset_sk_buff_head, GuessNotApplicable)
 				return t.setReadyState(mp)
-			} else { //nolint:revive // TODO(NET) Fix revive linter
+			} else {
```

Comment:
Shouldn't the `else` be removed?

Reply:
Yeah now that you mention it I see this was probably a return-else linter rule. It sure seems something was different about the linting methodology in the original PR cause I see no lint error here at all when I run it. There are multiple ways to invoke the linter so maybe they were using it differently than me.